### PR TITLE
Vision of Abyssea skills, ID, and scripts

### PR DESCRIPTION
**Pull request by R3P0FFXI**
due to incoming WOTG and TOAU skill commits started at skill id 900 so for now there will be gaps in the list

### CODE REVIEW

File: sql/mob_skill_lists.sql [hunk 1]
```diff
@@ -2616,3 +2616,392 @@ INSERT INTO `mob_skill_lists` VALUES ('Skirling_Liger',774,273);
...
+INSERT INTO `mob_skill_lists` VALUES ('Balaur_Pre_50',904,820); -- thunder breath
+
+INSERT INTO `mob_skill_lists` VALUES ('Balaur_Post_50',905,819); -- blizzard breath -- chance of ice spikes effect
+INSERT INTO `mob_skill_lists` VALUES ('Balaur_Post_50',905,820); -- thunder breath -- chance of shock spikes effect
+INSERT INTO `mob_skill_lists` VALUES ('Balaur_Post_50',905,821); -- radiant breath
+INSERT INTO `mob_skill_lists` VALUES ('Balaur_Post_50',905,822); -- chaos breath -- chance of dread spikes effect
+-- INSERT INTO `mob_skill_lists` VALUES ('Balaur_Post_50',905,2222); -- hurricane breath
+
+-- Bloodeye_Vileberry
+INSERT INTO `mob_skill_lists` VALUES ('Bloodeye_Vileberry',906,783); -- Words_of_Bane
```
Comment (TeoTwawki) [inline sql/mob_skill_lists.sql]: Are you sure there is no existing skill list with these exact skills included?

File: scripts/zones/Abyssea-Konschtat/mobs/Balaur.lua [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+function onMobDisengage(mob)
+end;
+
+-----------------------------------
+-- onMobFight
+-----------------------------------
+
+function onMobFight(mob,target)
+    -- Uncertain of threshold. Going with 50% for now.
+    -- (possibly varies, perhaps is simply lower HP = greater cast chance?)
```
Comment (R3P0FFXI) [inline scripts/zones/Abyssea-Konschtat/mobs/Balaur.lua]: @TeoTwawki this is the same situation at 50% HP the list should change

File: sql/mob_skill_lists.sql [hunk 3]
```diff
@@ -2616,3 +2616,392 @@ INSERT INTO `mob_skill_lists` VALUES ('Skirling_Liger',774,273);
...
+-- Alkonost
+INSERT INTO `mob_skill_lists` VALUES ('Alkonost',900,576); -- back heel
+-- INSERT INTO `mob_skill_lists` VALUES ('Alkonost',900,579); -- choke breath
+INSERT INTO `mob_skill_lists` VALUES ('Alkonost',900,578); -- nihlity song
+-- INSERT INTO `mob_skill_lists` VALUES ('Alkonost',900,580); -- fantod
+-- INSERT INTO `mob_skill_lists` VALUES ('Alkonost',900,1330); -- hoof volley
+INSERT INTO `mob_skill_lists` VALUES ('Alkonost',900,2828); -- jettatura
+
+-- Arimaspi
+INSERT INTO `mob_skill_lists` VALUES ('Arimaspi',901,548); -- blindeye
```
Comment (R3P0FFXI) [inline sql/mob_skill_lists.sql]: @TeoTwawki removed the custom one since it is the same list

**Comment by teschnei** [comment]
there's both a meteor skill and spell... chlevnik has a meteor skill, for
example

**Comment by R3P0FFXI** [comment]
@teschnei Yes Hadhayosh is Meteor Spell it was verified via youtube video and linked in the NM script I posted

**Comment by whasf** [comment]
needs a refresh and love

**Comment by R3P0FFXI** [comment]
Closing for now until TOAU is merged

**Pull request closed without merging**